convene-web/app/javascript/src/identity.js

```diff
@@ -0,0 +1,18 @@
+export default class Identity {
```

Comment:
I'm trying to decide between using `Profile` and `Identity` to describe the collection of name/pronouns/avatar. Curious what @anaulin / @cjoulain think?

Reply:
"Profile" feels like the more commonly used term for this kind of thing. But I'm not sure I have a strong opinion about it.